src/taipy/core/pipeline/pipeline.py

```diff
@@ -126,7 +114,6 @@ def data_nodes(self) -> Dict[str, DataNode]:
         return data_nodes
 
     @property  # type: ignore
```

Comment:
```suggestion
    @property
```
I think we can safely remove the `# type: ignore` here since there is only 1 decorator now.
Not sure though

Reply:
ah agree thanks!